### DISCUSSION TITLE
Drone collision with another actor

### DISCUSSION THREAD

**Discussion by kwanyongcho7** [microsoft/AirSim#3802] [Support Q&A]
I want to check how to drone act after collision, So I made sphere to move really fast like stone. 

However, When it hits the hovering drone, only I can see the collision count up, not something like drone falling down, moving...

Is there any way I can make it fall down or crash or move wrong after collided with external actor??

**Comment by zimmy87** [comment]
You can use the client.simGetCollisionInfo() API to check the last collision the drone encountered. The CollisionInfo object returned has a object_name and object_id which can be compared against your sphere actor. As for changing the drone's behavior after a collision, that would need to be a change in the flight controller you're using.

**Reply by kwanyongcho7** [reply]
How can I make the change in Flight Controller??
